### PR TITLE
Dt/e2e

### PR DESCRIPTION
**Pull request by davejrt**
Runs e2e tests in vagrant in every commit. These run asynchronously so as to not block other steps

### CODE REVIEW

File: enterprise/dev/ci/ci/pipeline-steps.go [hunk 1]
```diff
@@ -246,7 +246,14 @@ func triggerE2E(c Config, commonEnv map[string]string) func(*bk.Pipeline) {
 	// Run e2e tests for release branches
 	// We do not run e2e tests on other branches until we can make them reliable.
 	// See RFC 137: https://docs.google.com/document/d/14f7lwfToeT6t_vxnGsCuXqf3QcB5GRZ2Zoy6kYqBAIQ/edit
-	runE2E := c.releaseBranch || c.taggedRelease || c.isBextReleaseBranch || c.patch
+	runE2E := c.releaseBranch || c.taggedRelease || c.isBextReleaseBranch || c.patch || c.branch == "main"
```
Comment (uwedeportivo) [inline enterprise/dev/ci/ci/pipeline-steps.go:249]: is the comment above it still valid ?

File: test/Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+# # vi: set ft=ruby :
+# Specify minimum Vagrant version and Vagrant API version
+Vagrant.require_version '>= 1.6.0'
+VAGRANTFILE_API_VERSION = '2'.freeze
+# Require YAML module
+require 'yaml'
+
+# Read YAML file with box details
+servers = YAML.load_file('servers.yaml')
+# Create boxes
```
Comment (uwedeportivo) [inline test/Vagrantfile:11]: vagrant is ruby ? didn't know :-)

**Review by uwedeportivo — APPROVED** [review]
LGTM, very nice

File: enterprise/dev/ci/ci/pipeline-steps.go (hunk 1 above)
Comment (davejrt) [reply enterprise/dev/ci/ci/pipeline-steps.go:247]: Good call, sorted

**Pull request merged**